robottelo/hosts.py

```diff
@@ -253,6 +254,19 @@ def download_install_rpm(self, repo_url, package_name):
         if result.status != 0:
             raise ContentHostError(f'Failed to install {package_name} rpm.')
 
+    def download_repos(self, repo_name):
+        """Downloads the satellite or capsule repos on the machine
+        :param repo_name: satellite or capsule repo_name
+        """
+        repo_location = (
+            f'{settings.repos.ohsnap_repo_host}/api/releases/'
```

Comment:
```suggestion
            f'{settings.repos.ohsnap_host}/api/releases/'
```

?? 
There is either a mismatch in validator or here !